File: 485-max-consecutive-ones/485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        consec = 0
        max_consec = 0
        
        for val in nums:
            if val == 1:
                consec += 1
                if consec > max_consec:
                    max_consec = consec
            else:
                consec = 0
            
        
        return max_consec

File: 485-max-consecutive-ones/485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let consec = 0;
    let max_consec = consec;
    
    for (let i=0; i<nums.length; i++){
        if (nums[i] === 1){
            consec++;
            if(consec > max_consec){
                max_consec = consec;
            }
        } else {
            consec = 0;
        }
    }
    
    return max_consec;
};